modules/openvino_code/package.json

```diff
@@ -207,6 +208,40 @@
             "default": "false",
             "description": "When checked inline complention will be generated in streaming mode"
           },
+          "openvinoCode.fillInTheMiddleMode": {
+            "order": 4,
+            "type": "boolean",
+            "default": "false",
+            "description":
+              "When checked, text before (above) and after (below) the cursor will be used for completion generation. When unckecked, only text before (above) the cursor will be used."
+          },
+          "openvinoCode.startToken": {
+            "order": 7,
+            "type": "string",
+            "default": "<fim_prefix>",
```

Comment:
`deepseek-coder` has a different FIM tokens format:  `"<｜fim▁begin｜>"`, check [tokenizer.json](https://huggingface.co/deepseek-ai/deepseek-coder-1.3b-instruct/raw/main/tokenizer.json)

Reply:
I think it makes more sense to make a FIM feature a check box and to hardcode FIM tokens, as long as the model list is predefined.